### PR TITLE
[FEAT] Improve the pin-bot workflow 📌

### PR DESCRIPTION
**Pull request by NoahCardoza**
Listens to reactions to messages on the server, namely the 📌 emoji, and can pin and unpin the message depending on whether the 📌 was added or removed.

Added inline `/pin` and support for 📌 as a prefix. E.G.

```
/pin this message
📌 pin this one too
```

### CODE REVIEW

File: app/bot.py [hunk 1]
```diff
@@ -4,58 +4,162 @@
...
-    load_dotenv()
-    sys.stdout.flush()
-    bot.run(os.getenv("TOKEN"))
+reaction_emoji = '📌'
+
+prefix_map = {
+    reaction_emoji: ''
+}
+
+ingore_message_reactions_once = {
```
Comment (NoahCardoza) [inline app/bot.py:14]: This stuff could probably be better tracked with a class, but it works as-is. I'd be happy to refactor it at a later date when I get more free time if you'd like.

File: app/bot.py [hunk 2]
```diff
@@ -4,58 +4,162 @@
...
 
-def main():
-    load_dotenv()
-    sys.stdout.flush()
-    bot.run(os.getenv("TOKEN"))
+reaction_emoji = '📌'
+
+prefix_map = {
+    reaction_emoji: ''
+}
```
Comment (NoahCardoza) [inline app/bot.py:12]: I was aiming to allow this to support the other pin-like emojis, but after thinking about it, I think that’s might make things a lot more complicated, seeing as how you'd have to keep track of different reaction stacks. However, it was designed with that in mind.

**Comment by wrussell1999** [comment]
LGTM!

**Review by wrussell1999 — APPROVED** [review]
LGTM

**Pull request merged**